calico_cni_k8s_test.go

```diff
@@ -30,6 +32,8 @@ import (
 func init() {
 	// Create a random seed
 	rand.Seed(time.Now().UTC().UnixNano())
+	logrus.SetFormatter(&logutils.Formatter{})
```

Comment:
Same thing.  No context logger is called so its not necessary to instantiate the formatter.